### ISSUE TITLE
Path to download directly to network drive?

### ISSUE THREAD

**Issue by jtoobi** [vitiko98/qobuz-dl#115]
Is it possible to download files directly to a network drive? What would the path look like? Every version of a path I use ends up with a new folder being created on my internal hard drive.

**Comment by coreyoilar** [comment]
Yes.  Map the network drive with a drive letter.  Then in the configuration file under the location, indicate the drive letter followed by the path.  For example on my win10 computer, I have another computer mapped to drive W: so my path is W:\incoming\qobuz.  I don't have much experience on a mac but I can confirm by my experience on windows that this works as it should.  Just make sure that your mapped drive is allowed permissions to write to the network location.

**Comment by sramsay100** [comment]
Yes, this works and is currently what I am doing. Mapped netork drive references work perfectly fine on gthe Windows OS.
